src/wallabyTest.ts

```diff
@@ -17,3 +17,4 @@ TestBed.initTestEnvironment(
     BrowserDynamicTestingModule,
     platformBrowserDynamicTesting()
 );
+TestBed.initTestEnvironment = () => {};
```

Comment:
If it's VS Code complaining you may just ignore it, or do this to make it happy:
```javascript
TestBed.initTestEnvironment = (a: any, b: any): any => {};
```